tests/integration/scripteditor.ts

```diff
@@ -127,6 +146,42 @@ describe('Script Editor tests', () => {
     cy.closeTab(-1);
   });
 
+  // test to check if output kernel has expected output
+  it('checks for valid output', () => {
+    openFile('py');
+    cy.get('button[title="Run"]').click({ force: true });
+    cy.get('.elyra-ScriptEditor-OutputArea-output').should(
+      'have.text',
+      'Hello Elyra\n'
+    );
+
+    //close console tab
+    cy.closeTab(-1);
+
+    // Close editor tab
+    cy.closeTab(-1);
+  });
+
+  // test to check if output kernel has Error message for invalid code
+  it('checks for Error message', () => {
+    openFile('py');
+    cy.get('span[role="presentation"]').type('print("test")\n');
+    cy.get('button[title="Run"]').click({ force: true });
+    cy.get('.elyra-ScriptEditor-OutputArea-output').should(
+      'have.text',
+      'Error : SyntaxError - unterminated string literal (detected at line 1) (3615489043.py, line 1)'
+    );
```

Comment:
This will fail since the id created for the file name in the error log is can change depending on where it runs.
This block can be switched to `cy.findByText(/Error : SyntaxError/i).should('be.visible');`